plugins/AuthUI/src/commonMain/kotlin/io/github/temk0/supabase/authui/ui/GoogleButton.kt

```diff
@@ -0,0 +1,7 @@
+package io.github.temk0.supabase.authui.ui
+
+import androidx.compose.runtime.Composable
+import io.github.temk0.supabase.authui.AuthUI
+
+@Composable
+expect fun AuthUI.rememberLoginWithGoogle(onResult: (NativeSignInResult) -> Unit = {} , fallback: () -> Unit = {/*use gotrue.loginWith(Google) as default*/}): NativeSignInState
```

Comment:
```suggestion
expect fun AuthUI.rememberLoginWithGoogle(onResult: (NativeSignInResult) -> Unit = {} , fallback: () -> Unit = {/*use gotrue.loginWith(Google) as default*/}): NativeSignInState
```
The default value for fallback should be uncommented, I just put it into a comment because this wasn't valid code 👍